### PR TITLE
Provide copyright field in RSS feed

### PR DESCRIPTION
**Pull request by anderbubble**
I'm working on enhancing Nikola to support publishing a podcast RSS feed. Most of this work is going into a "postcast" plugin I'm working on, and will submit or publish eventually; but some of what I'm trying to do requires support in Nikola itself.

This PR aims to support providing a copyright field in RSS feeds [as recommended by iTunes connect](https://help.apple.com/itc/podcasts_connect/#/itcb54353390).

Other related PRs I'm working on:

* #2675 (explicit / static guids)
* #2674 (iTunes rss tags)

I'd also be interested in direction for where and how to publish a plugin, when that's ready.

### CODE REVIEW

File: nikola/nikola.py [hunk 1]
```diff
@@ -1660,6 +1660,11 @@ def generic_rss_renderer(self, lang, title, link, description, timeline, output_
             language=lang
         )
 
+        copyright = self.config['CONTENT_FOOTER'](lang)
```
Comment (Kwpolska) [inline nikola/nikola.py]: The existence of `copyright` should be configurable, and `CONTENT_FOOTER` might not be the right place to take it from.

**Comment by anderbubble** [comment]
First, thank you for reviewing this pull request!

I'm happy to address the requested changes. I'll change `copyright` to an argument for the relevant function.

Do you have a recommendation or a better general-purpose place to take that from?

**Comment by Kwpolska** [comment]
I’m afraid there isn’t anything immediately suitable. You could always create a new config variable — say, `RSS_COPYRIGHT`, and maybe `RSS_COPYRIGHT_PLAIN`.

**Comment by anderbubble** [comment]
@Kwpolska I think I've addressed your concerns. Let me know how this looks now. (I'm happy to squash the commits, too, if that's useful.)

File: nikola/conf.py.in [hunk 2]
```diff
@@ -1240,3 +1240,9 @@ GLOBAL_CONTEXT = {}
 # GLOBAL_CONTEXT as parameter when the template is about to be
 # rendered
 GLOBAL_CONTEXT_FILLER = []
+
+# A simple copyright tag for inclusion in RSS feeds that works just
```
Comment (Kwpolska) [inline nikola/conf.py.in]: Put it under `CONTENT_FOOTER`.

File: nikola/nikola.py [hunk 3]
```diff
@@ -1660,6 +1675,11 @@ def generic_rss_renderer(self, lang, title, link, description, timeline, output_
             language=lang
         )
 
+        if copyright_ is True:
```
Comment (Kwpolska) [inline nikola/nikola.py]: Just do `if copyright_:` here. Add a comment for future readers that the second `if` is checking if any copyright info is set.

**Comment by anderbubble** [comment]
The test for equivalency to True is intentionally functional: it allows you
to override the configured copyright by providing one explicitly as an
argument.

**Comment by anderbubble** [comment]
@Kwpolska I can rebase on top of current master if you'd like, to resolve the conflict; should I squash as well, or leave as-is?

**Comment by Kwpolska** [comment]
No need to squash.

**Comment by Kwpolska** [comment]
Merged. Thanks for contributing to Nikola! :tada:

**Pull request merged**